lib/client.js

```diff
@@ -58,41 +58,40 @@ Client.prototype.sftp = function(callback) {
   }
 
   var self = this;
-  var ssh = new Connection();
```

Comment:
```js
var ssh = this._ssh = new Connection();
```

Reply:
I prefer to use `var ssh = this._ssh = new Connection();` like @popomore suggested.